tsl/src/nodes/skip_scan/planner.c

```diff
@@ -1095,7 +1097,7 @@ sort_indexquals(IndexOptInfo *indexinfo, List *quals)
 {
 	List *indexclauses[INDEX_MAX_KEYS] = { 0 };
 	List *ordered_list = NIL;
-	int quals_len = list_length(quals);
+	int max_ind = 0;
```

Comment:
Is it the same index as `IndexOptInfo.ncolumns` or `nkeycolumns`?

Might be good to add a test with an index-only scan using a covering btree index, not sure if we have anything like this (`CREATE INDEX tab_x_y ON tab(x) INCLUDE (y);`)